tools/mo/openvino/tools/mo/main.py

```diff
@@ -66,7 +57,7 @@ def main(cli_parser: argparse.ArgumentParser, framework=None):
             for el in analysis_results.get_messages():
                 log.error(el, extra={'analysis_info': True})
         log.error(err)
-        if not argv['use_new_frontend'] and is_tf:
+        if hasattr(argv, 'use_new_frontend') and not argv.use_new_frontend and is_tf:
```

Comment:
is it mandatory to check the existence of argument in the dictionary? In the code above you don't check it:) May be `not argv.use_new_frontend and is_tf` is enough?

Reply:
In code above I don't check it because we have lines before try-catch block
`argv = cli_parser.parse_args()`
...
`argv = vars(argv)`
`argv = cli_parser.parse_args()` returns namespace with all mo params even if they are not set. Not set params have `None `values.
`argv = vars(argv)` converts Namespace to dictionary which has all parameters as well. 

But in new code we have  `argv = None`  before try-catch block.
If convert fails before returning parsed dictionary with params then argv remains with None. So in this case this check will cause error.